### PR TITLE
chore(monorepo): remove jest-styled-components

### PR DESCRIPTION
**Pull request by koji**
# Overview
remove jest-styled-components
https://opentrons.atlassian.net/browse/AUTH-82

# Test Plan
the following commands work as expect

- make check-js
- make lint-js
- make lint-css
- make test-js

# Changelog

# Review requests

# Risk assessment
low

### CODE REVIEW

File: __mocks__/electron-store.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // mock electron-store
 'use strict'
 import { vi } from 'vitest'
-import { DEFAULTS_V12, migrate } from '../app-shell-odd/src/config/migrate'
+// import { DEFAULTS_V12, migrate } from '../app-shell-odd/src/config/migrate'
```
Comment (jerader) [inline __mocks__/electron-store.js:4]: why is this commented out?

**Review by jerader — APPROVED** [review]
lgtm! thanks for removing this

File: __mocks__/electron-store.js (hunk 1 above)
Comment (koji) [reply __mocks__/electron-store.js:4]: The line will be cleared by another PR.

Comment (jerader) [reply __mocks__/electron-store.js:4]: sounds good, thanks for clarifying!

**Pull request merged**